svg-spritemap.js

```diff
@@ -14,28 +16,74 @@ function SVGSpritemapPlugin(options) {
         glob: {},
         prefix: 'sprite-',
         gutter: 2,
-        filename: 'spritemap.svg'
+        filename: 'spritemap.svg',
```

Comment:
That would make this BIC and this follows the setup used in plugins like [`extract-text-webpack-plugin`](https://github.com/webpack-contrib/extract-text-webpack-plugin#options).